src/js/sandbox/commands.js

```diff
@@ -15,12 +15,12 @@ var Warning = Errors.Warning;
 var CommandResult = Errors.CommandResult;
 
 var instantCommands = [
-  [/^ls/, function() {
+  [/^ls /, function() {
```

Comment:
It would be better to use `/^ls\b` instead, I think. `\b` in regex represents a word boundary, which is more explicit in meaning than a space.